.travis.yml

```diff
@@ -18,3 +20,5 @@ services:
   - docker
 before_install:
   - (test $RUN_ON_LIVE_SERVER = 1 && ./.travis/start_hawkular_services.sh -d && ./.travis/wait_for_services.rb) || echo "Skipping live server"
+after_success:
+  - ./.travis.docs.rb
```

Comment:
> I workaround it with a simple hack:
> https://github.com/hawkular/hawkular-client-ruby/pull/198/files#diff-515e88bb27e03f935a08a01e14f1d448R17

That seems to be good.

I was suggesting that one because of this: https://docs.travis-ci.com/user/deployment/rubygems/
We might be able to also use before/after_deploy and have it as part of our deploy pipeline. 
e.g. build docs, post to rubygems, have a beer.